File: src/create_wallets.py
from eth_account import Account
from eth_account.hdaccount import generate_mnemonic

Account.enable_unaudited_hdwallet_features()

mnemonic = generate_mnemonic(num_words=12, lang="english")
print(f"Mnemonic: {mnemonic}\n")

for i in range(10):
    acct = Account.from_mnemonic(mnemonic, account_path=f"m/44'/60'/0'/0/{i}")
    print(f"Address {i}: {acct.address}")

File: src/create_wallets.js
const { ethers } = require("ethers");

const mnemonic = ethers.Mnemonic.entropyToPhrase(ethers.randomBytes(16));
console.log(`Mnemonic: ${mnemonic}\n`);

for (let i = 0; i < 10; i++) {
  const wallet = ethers.HDNodeWallet.fromMnemonic(
    ethers.Mnemonic.fromPhrase(mnemonic),
    `m/44'/60'/0'/0/${i}`
  );
  console.log(`Address ${i}: ${wallet.address}`);
}
